huxley/core/admin/delegate.py

```diff
@@ -29,6 +30,23 @@ def roster(self, request):
 
         return roster
 
+    def load(self, request):
+        '''Loads new Assignments.'''
+        delegates = request.FILES
+        reader = csv.reader(delegates['csv'])
+
+        assignments = {}
+        for assignment in Assignment.objects.all():
+            assignments[assignment.committee.name.encode('ascii', 'ignore') + assignment.country.name.encode('ascii', 'ignore') + assignment.school.name] = assignment
```

Comment:
The string concatenation is pretty gross. Dictionaries can be keyed by tuples, let's just do that:
```python
assignments[
    assignments[assignment.committee.name.encode('ascii', 'ignore'),
    ...,
    ..., # Don't forget the trailing comma!
] = assignment
```